### ISSUE TITLE
Gif images missing

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#3039]
#### Issue

I have several warning messages on my log (Warn level) on images missing just by navigating in some of the main screens (using SuiteP theme).
I don't know where are they expected to be, searching the code no direct reference to them (only for svg and png versions on \themes\SuiteP\images\sidebar ), and I can't find those gif images on Github.
What is happening here?

#### Actual Behavior

Images are missing
```
02/02/17 20:22:02 [10232][1][WARN] Image Resource_Chart.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image Create.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image List_Maps.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image Quick_Radius_Map.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image Create_Security_Group.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image Create_Contact_Vcard.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image Schedule_Call.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image Schedule_Meeting.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image Create_Contact_Vcard.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image View_Create_Email_Templates.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image View_Email_Templates.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image View_Diagnostics.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image View_Process_Audit.gif not found
02/02/17 20:22:02 [10232][1][WARN] Image View_Project_Tasks.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image Create_Person_Form.gif not found
02/02/17 20:22:05 [10232][1][WARN] Image Today.gif not found
```

#### Steps to Reproduce

1. Just activate log in warn level

#### Your Environment

* SuiteCRM Version used: 7.8
* PHP 5.5.12, MySQL 5.6.17, Firefox, Windows 10, Wamp server, Apache 2.4.19

**Comment by samus-aran** [comment]
Looks like its a legacy way of obtaining the icons for menus.  These particular references have png/svg but they are also in .css (dunno if they overwrite anything).  But the solution would be to ensure the images are retrieved a standard way.

**Comment by horus68** [comment]
Also reported in Forum for 7.8.2 version
[WARN] Image Create.gif not found (1/1) - https://suitecrm.com/forum/suitecrm-7-0-discussion/10904-warn-image-create-gif-not-found#45134

**Comment by pgorod** [comment]
@Dillon-Brown I think you can close this one, it is solved in #4169.
